Zip/minizip/include/Minizip.h

```diff
@@ -0,0 +1,15 @@
+//
+//  Minizip.h
+//  Zip
+//
+//  Created by Xavi Matos on 1/29/19.
+//  Copyright © 2019 Roy Marmelstein. All rights reserved.
+//
+
+#ifndef Minizip_h
+#define Minizip_h
+
+#import "unzip.h"
+#import "zip.h"
+
```

Comment:
An `#import "crypt.h"` is missing here.
Compiling also warns about it:
```
<module-includes>:1:1: warning: umbrella header for module 'Minizip' does not include header 'crypt.h'
#include "/Zip/Zip/minizip/include/Minizip.h"
^
```

Reply:
Hey Florian, I addressed that warning, but in the process came across a few more things worth addressing.
Most notably, I overwrote the project with the results of `swift package generate-xcodeproj` because the project wasn't building via Xcode due to the removal of the `module.modulemap` file.
This may endanger the desirability of the original SPM support PR back into the root **marmelroy/Zip** repo, so you may want to reconsider my PR.
I've made the changes in a series of commits so you may want to cherry-pick whichever bits you like.
¡Thanks for the feedback!

Reply:
@CalQL8ed-K-OS Yea, overwriting the Xcode project isn't a good idea for non-SPM projects. 🙂 
I'll check if I can manually integrate your changes. Maybe I can just tell SPM to ignore the module map.